modules/sfp_shodan.py

```diff
@@ -120,6 +135,18 @@ def handleEvent(self, event):
             evt = SpiderFootEvent("SEARCH_ENGINE_WEB_CONTENT", str(hosts), self.__name__, event)
             self.notifyListeners(evt)
 
+        if eventName == 'WEB_ANALYTICS_ID':
+            network = eventData.split(": ")[0]
+            analytics_id = eventData.split(": ")[1]
+            rec = self.searchHtml(analytics_id)
+
+            if rec is None:
+                return None
+
+            evt = SpiderFootEvent("SEARCH_ENGINE_WEB_CONTENT", str(rec), self.__name__, event)
```

Comment:
Can we return the content instead of wrapping it in `str()`? Doubt that would be unicode safe.

Reply:
Should the `'DOMAIN_NAME'` block also be updated?

This block is essentially copy pasta of existing code which also uses `str()`:


```py
        if eventName == "DOMAIN_NAME":
            hosts = self.searchHosts(eventData)
            if hosts is None:
                return None
            
            evt = SpiderFootEvent("SEARCH_ENGINE_WEB_CONTENT", str(hosts), self.__name__, event)
            self.notifyListeners(evt)

        if eventName == 'WEB_ANALYTICS_ID':
            network = eventData.split(": ")[0]
            analytics_id = eventData.split(": ")[1]
            rec = self.searchHtml(analytics_id)

            if rec is None:
                return None

            evt = SpiderFootEvent("SEARCH_ENGINE_WEB_CONTENT", str(rec), self.__name__, event)
            self.notifyListeners(evt)
            return None
```